### PR TITLE
Updated README data area instructions

### LINKED ISSUE #3212: Instructions to download data on README are wrong.
**Description**

I followed the instructions on the github page (https://github.com/USGS-Astrogeology/ISIS3#full-isis3-data-download).  It dumped data into my home area.  I looked around and inferred that I should follow part of the instructions for downloading all the data and copy-pasted the command given in the instructions (_cd $ISIS3DATA_).  Then I tried to update the TGO data, and copy-pasted the command given in the instructions (_rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/tgo data/_).  Now I have new data in a new folder where ISIS3 won't find it.  No error messages or anything to tell the user there is a problem.  I only know there is a problem because I spent months debugging my student's ISIS3 installation and know what to look for.  

**Suggested Edit**

I already suggested a fix but that didn't seem to help.  At this point I am suggesting you write out the explicit single-line rsync command that will put the data in the right folder on the user's computer.  

Thanks!

### PR DESCRIPTION
**Pull request by jessemapel**
I ported the changes to the website install guide to the README

## Description

All of the mission specific data download code snippets now start with `cd` to the data area and no longer assume the data area is called `data`. I also fixed several broken links.

## Related Issue

Fixes #3212 

## Motivation and Context

Previous instructions were ambiguous and users have reported downloading files into the wrong area.

## How Has This Been Tested?

Documentation renders on my [fork](https://github.com/jessemapel/ISIS3/tree/readme_install)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,240 +118,228 @@ To build and compile ISIS3 requires following the instructions listed below, whi
...
-Rolo Mission (kernels can be excluded):
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/newhorizons .
 
-<span id="RoloMission"></span>
 
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/rolo data/
+**Odyssey Mission (kernels can be excluded):**
 
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/odyssey .
```
Comment (krlberry) [inline README.md]: This one is missing a `cd $ISIS3DATA`

**Review by krlberry — CHANGES_REQUESTED** [review]
:+1: This looks good -- just found one mission data section missing the `cd $ISIS3DATA`

File: README.md (hunk 1 above)
Comment (krlberry) [reply README.md]: And actually, "Mars Odyssey" is already on this list earlier -- maybe this whole section should just be removed?

Comment (jessemapel) [reply README.md]: Should be fixed now

Comment (krlberry) [reply README.md]: @jessemapel :+1:

**Review by krlberry — APPROVED** [review]

**Pull request merged**